js/imageViewerAnnotate.js

```diff
@@ -182,12 +200,20 @@ IVEM.insertPreview = function (field, params) {
   if (params.hasOwnProperty("container_id")) {
     $container = $("div[data-ivem-container=" + params.container_id + "]");
     if ($container.length == 0) {
+      // Container styling is based on portrait and landscape image use cases
       $container = $("<div></div>")
         .attr("data-ivem-container", params.container_id)
         .css("position", "relative")
-        .css("padding-top", "30px")
-        .css("margin-left", "auto")
-        .css("margin-right", "auto");
+        .css("margin", "5px auto 40px")
+        .css("max-width", "max(70vw, 250px)");
```

Comment:
> we only want to add the `max-width` if the image is portrait, so that needs to be checked somehow

See [this](https://github.com/brown-ccv/redcap-image-annotator/pull/29/commits/28612bce710a7a12e5b42d6777d61f4d0cc2a975) commit

Reply:
For the max width styling, it seems `80vw` works well for phone/tablet orientations